<launch>

  <arg name="robot_id" default="$(optenv ROBOT_ID unnamed_robot)"/>



  <group ns="$(arg robot_id)">

    <node pkg="sdpo_ratf_driver" type="sdpo_ratf_driver_node"
        name="sdpo_ratf_driver" clear_params="true"
        output="$(optenv ROBOT_OUTPUT screen)">

      <param name="encoder_res" value="64.0"/>
      <param name="gear_reduction" value="43.8"/>
      <param name="serial_port_name" value="/dev/ttyACM0"/>

    </node>

  </group>

</launch>
